package.nls.json

```diff
@@ -1,3 +1,5 @@
 {
-    "taipy.gui.md.generate.title": "Generate Taipy GUI element"
+    "taipy.gui.md.generate.title": "Generate Taipy GUI element",
```

Comment:
Generate Taipy GUI element
if it appears in the 'Taipy' category, I would rephrase that to "Generate visual element"

Reply:
It does not, but we can try this `Taipy Studio Gui Helper: Generate Visual Element`